NETEZZA.md

```diff
@@ -28,12 +28,19 @@ It consists of 2 VMware images which theoretically could be run on any VMware en
    * Test if the Netezza Emulator is visible from the development vm (example: $ping 192.168.85.128)
    * Optionally install the linux client tools on the VM.
 
-3. Create the database NETEZZATEST using either the Administrator GUI or nzsql 
+3. Configure the Netezza connection for the DbFit acceptance tests in the file 'dbfit-java\TestDbConnectionDbFitNetezza.properties.custom':
```

Comment:
What do you think about using ` instead of ' quotes here for sake of formatting enhancement?